base/doc/ltnews42.tex

```diff
@@ -155,6 +155,23 @@ \subsection{Normalizing key names for block environments}
 
 \section{New or improved commands}
 
+\subsection{Support separate font families for script fonts}
+
+In \TeX's math processing separate fonts can be selected for text, script and
+scriptscript sizes. \LaTeX's NFSS traditionally uses this to load three sizes
+of the same font, handling adjustment needed for making fonts appear better in
+a script location through the use of optical sizes. This works great for traditional
```

Comment:
\LaTeX's NFSS traditionally uses the same font family at different sizes, handling ...